picard/tagger.py

```diff
@@ -361,13 +371,16 @@ def add_files(self, filenames, target=None):
                     self.files[filename] = file
                     new_files.append(file)
         if new_files:
+            self.count_file = 0
+            self.total_file = len(new_files)
             log.debug("Adding files %r", new_files)
             new_files.sort(key=lambda x: x.filename)
             if target is None or target is self.unmatched_files:
                 self.unmatched_files.add_files(new_files)
                 target = None
             for file in new_files:
-                file.load(partial(self._file_loaded, target=target))
+                self.count_file += 1
```

Comment:
`self.count_file` will be equal to `len(new_files)` at end of the loop or i miss smt ?